src/sinol_make/config/__init__.py

```diff
@@ -0,0 +1,39 @@
+import yaml, os, subprocess
+
+class Config:
+	def __init__(self):
+		config_path = os.path.join(os.path.dirname(__file__), 'config.yml')
+		if not os.path.exists(config_path):
+			open(config_path, 'w').write(yaml.dump({}))
+			self.setup()
```

Comment:
I would say that the default behavior should be to panic -- our (current) assumption should be that the user is working on an existing repository with the task. It will change after adding a command for creating a directory with the task, but then still the tool should assume that the config exists.

Reply:
This config is for global settings, for example the names of compiler programs etc. Maybe we should rename the file to GlobalConfig or something like that?

Reply:
Yes, I wanted to store there `c_compiler`, `cpp_compiler`, ... and maybe other things in the future

Reply:
Having multiple config files might be harmful -- one might forget where one was or what is its content, the configs maybe would have to be copy-pasted between task packages, it pollutes the file structure with not relevant files for the task, etc. It also slows down the first usage of the tool -- one has to write a bunch of config values before running it for the first time. Thus we have to be careful with creating config files. Is it really needed? If someone would want to temporarily overwrite a config value, maybe a commandline argument with sensible default values might be better? Will we really have the need to put in this config other kind of values?

Reply:
Removed global config